content/en/docs/concepts/services-networking/service.md

```diff
@@ -188,9 +188,10 @@ selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
 ### Over Capacity Endpoints
-If an Endpoints resource has more than 1000 endpoints then a Kubernetes v1.21 (or later)
```

Comment:
We should maybe edit the v1.21 docs (either now or post release) to remove the “or later” part from the v1.21 docs. WDYT @swetharepakula ?

Reply:
Oh that is a good point! I can send in a followup pr for that. Would branch would that pr need to be against?

Reply:
At the moment, that is the _main_ branch: commits there are continuously published to https://kubernetes.io/

Reply:
Corresponding PR: https://github.com/kubernetes/website/pull/29122